Scotty.sh

```
@@ -0,0 +1,90 @@
+}
+
+# @task on:remote
+updateSymlinks() {
+    rm -rf $NEW_RELEASE_DIR/storage
+    cd $NEW_RELEASE_DIR
+    ln -nfs $PERSISTENT_DIR/storage storage
+}
+
+# @task on:remote
+migrateDatabase() {
+    cd $NEW_RELEASE_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)